requirements.txt

```diff
@@ -1 +1,3 @@
 Django~=2.2.2
+pylint==2.3.1
```

Comment:
These aren't runtime deps. Create a separate `test-requirements.txt` file for them. They are only used for testing.

Reply:
I have to create a file test-requirements.txt 
and put it there
Django~=2.2.2
pylint==2.3.1 
Right?

Reply:
No. Don't put django there.

Reply:
Why use `~=` with Django is a good idea, but use `pylint~=2.3.1` is not a good idea? @webknjaz

Reply:
It is not about the version pinning. It's about mangling runtime requirements with test requirements.

Reply:
Sorry, @webknjaz I don't understand why need `~=` if exist [dependabot](https://dependabot.com).

Reply:
Don't confuse lib/dev deps (open ended) and app/test/env deps (pinned).